src/Core/src/Handlers/WebView/WebViewHandler.Windows.cs

```diff
@@ -24,21 +26,54 @@ internal WebNavigationEvent CurrentNavigationEvent
 		protected override void ConnectHandler(WebView2 platformView)
 		{
 			platformView.CoreWebView2Initialized += OnCoreWebView2Initialized;
-
 			base.ConnectHandler(platformView);
+
+			if (platformView.IsLoaded)
+				OnLoaded();
+			else
+				platformView.Loaded += OnWebViewLoaded;
 		}
 
-		protected override void DisconnectHandler(WebView2 platformView)
+		void OnWebViewLoaded(object sender, UI.Xaml.RoutedEventArgs e)
+		{
+			OnLoaded();
+		}
+
+		void OnLoaded()
+		{
+			_window = MauiContext!.GetPlatformWindow();
+			_window.Closed += OnWindowClosed;
```

Comment:
`_window` can be `null` here (per the line above, and per the field declaration), no?

Reply:
Oh wait nevermind. I read the code wrong. Ignore 😄